CONDUCT.md

```diff
@@ -0,0 +1,84 @@
+* sexual images in public spaces
+* deliberate intimidation, stalking, following
+* harassing photography or recording
+* sustained disruption of talks or other events
+* inappropriate physical contact
+* unwelcome sexual attention
+* advocating for, or encouraging, any of the above behaviour
+
+Sexual language and imagery is not appropriate for Our Networks presentations and interactions. 
+
+Participants asked to stop any harassing behavior are expected to comply immediately. Participants violating these rules may be sanctioned or asked to leave, at the discretion of the organizers.
+
+## Scope
+
+This Code of Conduct applies both at the event and in public spaces when an individual is representing Our Networks.
+
+## Enforcement
+
+At events, Our Networks organizers will identify themselves and any incidents can be reported to them directly. Event organizers will be happy to help participants by providing escorts, contacting local law enforcement, or otherwise assisting those experiencing harassment to feel safe for the duration of an event. We value your participation.
+
+If a participant engages in harassing behaviour, the organizers may take any action they deem appropriate, including warning the offender or expulsion from events and online forums.
+
+If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact an organizer immediately.
+
+Those who wish to report a violation but don't feel comfortable talking to the organizers online or at an event can email [coc@ournetworks.ca](mailto:coc@ournetworks.ca), which is **currently** monitored by [**Dawn, @dcwalk**](http://dcwalker.ca), and will be monitored during the event on an active basis.
+
+Our [internal guidelines](./CONDUCT.md) for enforcing our Code of Conduct are viewable online.
+
+_These are adapted from the [Toronto Mesh Code of Conduct](https://tomesh.net/code-of-conduct/), which in turn was based on the anti-harassment policy from the [Geek Feminism Wiki](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy) and draws inspiration from [No more rock stars: how to stop abuse in tech communities](https://hypatia.ca/2016/06/21/no-more-rock-stars/). The section about expected behaviour is adapted from the [Gamma Space Code of Conduct](https://github.com/GammaSpace/policies/blob/master/code-of-conduct.md)._
+
+---
+
+## Guidelines
+
+Our Networks adopted the [Toronto Mesh Code of Conduct](https://tomesh.net/code-of-conduct/) for our January 14, 2017 event, and consensed to actively use a conference-specific code of conduct on [XXXX, 2018](https://github.com/ournetworks/2018/pull/33). The Code of Conduct language is in our [2018 organizing](./CONDUCT.md) repository and on our website at [ournetworks.ca/code-of-conduct/](https://ournetworks.ca/code-of-conduct).
```

Comment:
`[2018 organizing](./CONDUCT.md)`: Point to the top-level of the organizing repository?

Reply:
hmmm, I'm trying to link directly to the coc in each place (website, repo) so I'm inclined to leave as is? Does that make sense?

Reply:
yeah. makes sense